collective/quickupload/browser/quickupload_settings.py

```diff
@@ -106,6 +106,18 @@ class IQuickUploadControlPanel(Interface):
         default=2,
         required=True)
 
+    object_unique_id = Bool(
+        title=_(u"title_object_unique_id", default=u"Choose unique file id"),
+        description=_(
+            u"description_object_unique_id",
+            default=u"Choose a unique id for each uploaded file."
+                    u"Checked: Your files will never be overridden and you'll "
```

Comment:
hmm... I choose the same syntax like in the `object_override`-field description...